schemas/common_definitions.json

```diff
@@ -51,6 +54,15 @@
         "description": {
           "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
         },
+        "guidance": {
```

Comment:
Content blocks are for generic content. The guidance should be a controlled structure. It won't make sense for us to allow guidance for all content blocks. This makes sense as part of primary content in introduction. Same way we have allowed guidance for question.

Reply:
Ok, moved it to `primary_content` so nesting removed and guidance not part of generic content anymore.